### PR TITLE
Review Quest Lines 400-700

### PR DESCRIPTION
**Pull request by ttgmichael**
phew! Proofread lines from Quest file to see what RTT may have missed. Verified translations with Korean text.

Let me know whether you see any lines that was missed. :+1:

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -399,302 +399,302 @@ QUEST_20150317_000398	{memo X}The first Paladin was told many things by the Godd
...
-QUEST_20150317_000661	I came to Agailla Flurry's room to pick up cleaning materials, {nl}and I saw Flurry talking to a woman that glowed like a huge flame.
-QUEST_20150317_000662	The sight terrified me, so I hid behind a wall and watched. {nl}Then suddenly, a scary demon rushed in.
+QUEST_20150317_000654	$Agailla Flurry's Diary
+QUEST_20150317_000655	$Lord Helgasercle is still going after my tower, {nl}and I can't find a successor to continue my mission.
+QUEST_20150317_000656	$My life feels like that of a waning fire. {nl}Even so, I shall light up the last bit of my life to find one.
+QUEST_20150317_000657	$Because of Helgasercle, a great chaos will come down on this world. {nl}It's a desperate gamble to leave the tower to a Goddess who should be taking care of the world's flame.
+QUEST_20150317_000658	$But my mission must not fall under the hands of the demons, {nl}so I am counting on the future Goddess Laima foretold.
+QUEST_20150317_000659	$It was such a long journey. {nl}We'll know whether or not it was the right life or decision when the time comes.
+QUEST_20150317_000660	$The Diary of Jane, the Probationary Wizard
```
Comment (Sunaries) [inline QUEST.tsv]: Probationary / Trainee / Apprentice Wizard Jane's Diary

Three options. One choice.

File: QUEST.tsv [hunk 2]
```diff
@@ -399,302 +399,302 @@ QUEST_20150317_000398	{memo X}The first Paladin was told many things by the Godd
...
-QUEST_20150317_000693	This object must be from the era of Agailla Flurry as no one knows the existence of this object.
-QUEST_20150317_000694	I am sure that this object will bring a new beginning of magic, {nl}but since it is separated at the highest place atop the tower, we have no way to reach it.
-QUEST_20150317_000695	We can't demolish the tower nor destroy part of it just to get it.
-QUEST_20150317_000696	I will end my writing by stating that this object has more to be investigated.
-QUEST_20150317_000697	Potter's Clay Tablet
+QUEST_20150317_000664	$The demon somehow disappeared behind the flaming woman... {nl}There was nothing I could do, so I ran away.
+QUEST_20150317_000665	$After a few days, my seniors told me that Gabija would be the new owner of the tower. {nl}When the Goddesses disappeared from the world, I was told to keep quiet.
+QUEST_20150317_000666	$Are things really going to be okay? {nl}Well... those people were never wrong about anything before so I guess it will be.
+QUEST_20150317_000667	$Chronomancer Ron
+QUEST_20150317_000668	$Arch Mage Agailla Flurry, owner of the Mage Tower, {nl}made many contributions to magic.
```
Comment (Sunaries) [inline QUEST.tsv]: Arch Mage -> Archmage ?

File: QUEST.tsv (hunk 1 above)
Comment (ttgmichael) [inline QUEST.tsv]: Chose Apprentice! :)

File: QUEST.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST.tsv]: yep!

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**